opensearch-operator/pkg/helpers/helpers.go

```diff
@@ -244,3 +244,15 @@ func HasDataRole(nodePool *opsterv1.NodePool) bool {
 func HasManagerRole(nodePool *opsterv1.NodePool) bool {
 	return ContainsString(nodePool.Roles, "master") || ContainsString(nodePool.Roles, "cluster_manager")
 }
+
+// Compares whether v1 is LessThan v2
+func CompareVersions(v1 string, v2 string) bool {
+	lessThan := false
+	ver1, err := version.NewVersion(v1)
+
+	ver2, _ := version.NewVersion(v2)
+	if err == nil && ver1.LessThan(ver2) {
+		lessThan = true
+	}
+	return lessThan
```

Comment:
Can be written shorter:
```suggestion
	return err == nil && ver1.LessThan(ver2)
```

Reply:
Hi @swoehrl-mw, I have made the changes requested, please let me know if there is anything else